File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-custom-xml-parts-to-documents-by-using-vsto-add-ins_3.cs
        private void AddCustomXmlPartToPresentation(PowerPoint.Presentation presentation)
        {
            string xmlString =
                "<?xml version=\"1.0\" encoding=\"utf-8\" ?>" +
                "<employees xmlns=\"http://schemas.microsoft.com/vsto/samples\">" +
                    "<employee>" +
                        "<name>Karina Leal</name>" +
                        "<hireDate>1999-04-01</hireDate>" +
                        "<title>Manager</title>" +
                    "</employee>" +
                "</employees>";

            Office.CustomXMLPart employeeXMLPart = 
                presentation.CustomXMLParts.Add(xmlString, missing);
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-custom-xml-parts-to-documents-by-using-vsto-add-ins_3.vb
    Private Sub AddCustomXmlPartToPresentation(ByVal presentation As PowerPoint.Presentation)
        Dim xmlString As String = _
            "<?xml version=""1.0"" encoding=""utf-8"" ?>" & _
                "<employees xmlns=""http://schemas.microsoft.com/vsto/samples"">" & _
                    "<employee>" & _
                        "<name>Karina Leal</name>" & _
                        "<hireDate>1999-04-01</hireDate>" & _
                        "<title>Manager</title>" & _
                    "</employee>" & _
                "</employees>"
        Dim employeeXMLPart As Office.CustomXMLPart = _
            presentation.CustomXMLParts.Add(xmlString)
    End Sub